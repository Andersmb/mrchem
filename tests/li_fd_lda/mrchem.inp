# vim:syntax=sh:

rel_prec = 1.0e-4                 # Overall relative precision

MRA {
  order = 6                       # Polynomial order
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
charge = 0
multiplicity = 2
angstrom = true
$coords
Li      0.0000  0.0000  0.0000
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
  restricted = false
}

DFT {
  spin = true
  $functionals
  LDA
  $end
  density_cutoff = 1.0e-10
}

ExternalField {
electric_run = true
electric_field = [ 0.0, 0.0, 0.001 ]
}

Properties {
  scf_energy = true               # Compute ground state energy
  dipole_moment = true            # Compute electrostatic dipole moment
  polarizability = false
}

SCF {
  run = true
  orbital_thrs = 1.0e-3           # Convergence threshold in orbital residual
  kain = 3                        # Length of KAIN iterative history
  initial_guess = SAD_DZ
  rotation = 5                    # Diagonalize Fock matrix every n'th iteration
  canonical = false               # Use canonical of localized orbitals
  write_orbitals = false
}

