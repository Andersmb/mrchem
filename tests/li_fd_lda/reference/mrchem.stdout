


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. b25d783)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : bd40f63

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 6
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -32.000000  -32.000000  -32.000000 ]
 upper bounds     = [   32.000000   32.000000   32.000000 ]
 total length     = [   64.000000   64.000000   64.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-03   2.61594e-03
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-03   2.61594e-03
============================================================


============================================================
              Projecting Gaussian-type density
------------------------------------------------------------
 Nr  Element                                 Rho_i
------------------------------------------------------------
  0     Li                            3.000000000000020e+00
------------------------------------------------------------
                 Wall time: 1.51589e+01 sec 
============================================================


============================================================
                  Projecting Hydrogen AOs
------------------------------------------------------------
    N    Atom   Label                     SquareNorm
------------------------------------------------------------
    1    Li1     1s                   1.000000184429093e+00
    2    Li1     2s                   1.000000367857642e+00
    3    Li1     2p                   9.999999910484429e-01
    4    Li1     2p                   9.999999910484429e-01
    5    Li1     2p                   9.999999910484429e-01
    6    Li1     3s                   1.000000109033399e+00
    7    Li1     3p                   9.999999968389940e-01
    8    Li1     3p                   9.999999968389940e-01
    9    Li1     3p                   9.999999968389941e-01
------------------------------------------------------------
                 Wall time: 9.45433e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Nuclear potential                     904      3.77027e-01
 Coulomb potential                     200      2.02178e+00
 XC evaluate xcfun                    1368      1.50749e+00
 XC energy                             456      2.57640e-05
 XC potential                          912      2.49374e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 3.93556e+00 sec 
============================================================


============================================================
                  Diagonalize Fock matrix
------------------------------------------------------------
 Compute Fock matrix                            6.68985e+00
 Diagonalize Fock matrix                        4.31077e-04
 Rotate orbitals                                4.02996e-02
 Compute Fock matrix                            7.83411e+00
 Diagonalize Fock matrix                        4.33157e-04
 Rotate orbitals                                2.31030e-02
------------------------------------------------------------
                 Wall time: 1.46072e+01 sec 
============================================================


============================================================
 OrbitalVector:   3 orbitals     3 occupied     3 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       1.000000000000e+00    a   1    1.00000e+00
   1     0       9.999999996030e-01    a   1    1.00000e+00
   2     0       1.000000000000e+00    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     904      3.27475e-01
 Coulomb density                       456      3.80870e-01
 Coulomb potential                     200      2.08720e+00
 XC alpha density                      456      2.61291e-01
 XC beta density                       456      1.41079e-01
 XC evaluate xcfun                    1368      1.43932e+00
 XC energy                             456      2.88610e-05
 XC potential                          912      2.65804e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 4.67434e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         7.248776120000000e-01
 Potential part                       2.636631093000000e+00
------------------------------------------------------------
                 Wall time: 3.36160e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        4.96763e-03
 Diagonalizing matrix                           8.26800e-05
------------------------------------------------------------
                 Wall time: 7.52971e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       2.756106540907633e-15
 Electronic energy                   -7.297861382726143e+00
------------------------------------------------------------
 Total energy                        -7.297861382726141e+00
------------------------------------------------------------
                 Wall time: 2.64362e+00 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -1.81781e+00
 Creating operator with lambda                 -1.13387e-01
 Creating operator with lambda                 -1.80266e+00
------------------------------------------------------------
                 Wall time: 4.35924e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 2.58592e+00
 Matrix part                                    4.03030e-05
------------------------------------------------------------
   0       1.59791e+00    456            0.00000e+00      0
   1       3.07857e-01    392            0.00000e+00      0
   2       1.57062e+00    456            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 2.62481e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.02804e+00   392    0.00000e+00     0   4.26194e+00
  1    1.00777e+00   264    0.00000e+00     0   3.42254e+00
  2    1.02063e+00   392    0.00000e+00     0   4.44495e+00
------------------------------------------------------------
                 Wall time: 1.21295e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.72800e-05
 Copy orbitals                                  2.18809e-02
 Copy orbital updates                           3.34064e-02
------------------------------------------------------------
                 Wall time: 5.53998e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     904      3.29839e-01
 Coulomb density                       456      3.99468e-01
 Coulomb potential                     200      2.08707e+00
 XC alpha density                      456      3.03538e-01
 XC beta density                       456      1.51152e-01
 XC evaluate xcfun                    1368      1.52098e+00
 XC energy                             456      2.96970e-05
 XC potential                          912      2.69037e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 4.82982e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         6.899619920000000e-01
 Potential part                       2.414516566000000e+00
------------------------------------------------------------
                 Wall time: 3.10456e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.87181e+00   5.46360e-02        392    a    1    0
  1  -1.17102e-01   5.91898e-02        264    a    1    0
  2  -1.86293e+00   6.55715e-02        392    b    1    0
------------------------------------------------------------
 Total error:                            1.03866e-01    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -3.733855213997970e+00    -3.73386e+00    1
 Kinetic       7.917843877468131e+00     7.91784e+00    1
 N-E          -1.760243131882017e+01    -1.76024e+01    1
 Coulomb       4.104980677934867e+00     4.10498e+00    1
 Exchange      0.000000000000000e+00     0.00000e+00    1
 X-C          -1.718254619308969e+00    -1.71825e+00    1
------------------------------------------------------------
 Electronic   -7.297861382726143e+00    -7.29786e+00    1
 Nuclear       2.756106540907633e-15     2.75611e-15    1
------------------------------------------------------------
 Total        -7.297861382726141e+00    -7.29786e+00    1
============================================================


################ Wall time: 2.61207e+01 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        4.39667e-03
 Diagonalizing matrix                           7.01340e-05
------------------------------------------------------------
                 Wall time: 6.74359e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       2.756106540907633e-15
 Electronic energy                   -7.341817902418930e+00
------------------------------------------------------------
 Total energy                        -7.341817902418927e+00
------------------------------------------------------------
                 Wall time: 2.64853e+00 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -1.87207e+00
 Creating operator with lambda                 -1.16845e-01
 Creating operator with lambda                 -1.86293e+00
------------------------------------------------------------
                 Wall time: 4.21338e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 2.41554e+00
 Matrix part                                    3.65200e-05
------------------------------------------------------------
   0       1.54377e+00    456            0.00000e+00      0
   1       2.70950e-01    328            0.00000e+00      0
   2       1.53083e+00    456            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 2.45313e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00590e+00   392    0.00000e+00     0   4.33458e+00
  1    9.98801e-01   264    0.00000e+00     0   3.70199e+00
  2    1.00290e+00   392    0.00000e+00     0   4.58451e+00
------------------------------------------------------------
                 Wall time: 1.26212e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.41770e-05
 Copy orbitals                                  2.49030e-02
 Copy orbital updates                           2.95174e-02
------------------------------------------------------------
                 Wall time: 5.45033e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     904      3.36512e-01
 Coulomb density                       456      4.17244e-01
 Coulomb potential                     200      2.31196e+00
 XC alpha density                      456      2.76291e-01
 XC beta density                       456      1.40995e-01
 XC evaluate xcfun                    1368      1.53384e+00
 XC energy                             456      4.16130e-05
 XC potential                          912      2.99352e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 5.05953e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         7.294787730000000e-01
 Potential part                       2.216876705000000e+00
------------------------------------------------------------
                 Wall time: 2.94643e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.87517e+00   8.28396e-03        392    a    1    1
  1  -1.16517e-01   1.50977e-02        264    a    1    0
  2  -1.86695e+00   9.81915e-03        392    b    1    1
------------------------------------------------------------
 Total error:                            1.98237e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -3.851836506578120e+00    -1.17981e-01    1
 Kinetic       7.388040700236935e+00    -5.29803e-01    1
 N-E          -1.707295511967196e+01     5.29476e-01    1
 Coulomb       4.016308370586184e+00    -8.86723e-02    1
 Exchange      0.000000000000000e+00     0.00000e+00    1
 X-C          -1.673155987720840e+00     4.50986e-02    1
------------------------------------------------------------
 Electronic   -7.341817902418930e+00    -4.39565e-02    1
 Nuclear       2.756106540907633e-15     0.00000e+00    1
------------------------------------------------------------
 Total        -7.341817902418927e+00    -4.39565e-02    1
============================================================


################ Wall time: 2.64988e+01 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       2.756106540907633e-15
 Electronic energy                   -7.343537463603601e+00
------------------------------------------------------------
 Total energy                        -7.343537463603599e+00
------------------------------------------------------------
                 Wall time: 2.25357e+00 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -1.87517e+00
 Creating operator with lambda                 -1.16517e-01
 Creating operator with lambda                 -1.86695e+00
------------------------------------------------------------
                 Wall time: 3.78288e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 2.16616e+00
 Matrix part                                    3.46888e-02
------------------------------------------------------------
   0       1.52792e+00    456            0.00000e+00      0
   1       2.70600e-01    392            0.00000e+00      0
   2       1.52145e+00    456            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 2.24975e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00198e+00   392    0.00000e+00     0   3.66547e+00
  1    1.00021e+00   264    0.00000e+00     0   3.11702e+00
  2    1.00082e+00   392    0.00000e+00     0   3.64627e+00
------------------------------------------------------------
                 Wall time: 1.04296e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.25930e-05
 Setup linear system                            5.25089e-02
 Solve linear system                            7.22580e-05
 Expand solution                                8.65199e-02
------------------------------------------------------------
                 Wall time: 1.39194e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     904      2.89984e-01
 Coulomb density                       456      3.54290e-01
 Coulomb potential                     200      1.90783e+00
 XC alpha density                      456      2.49395e-01
 XC beta density                       456      1.35336e-01
 XC evaluate xcfun                    1368      1.35782e+00
 XC energy                             456      2.64680e-05
 XC potential                          912      2.49407e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 4.32966e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         6.536485330000000e-01
 Potential part                       2.286887251000000e+00
------------------------------------------------------------
                 Wall time: 2.94061e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.87569e+00   3.20756e-03        392    a    1    1
  1  -1.16114e-01   9.47537e-03        264    a    1    1
  2  -1.86793e+00   2.58593e-03        392    b    1    1
------------------------------------------------------------
 Total error:                            1.03324e-02    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -3.858637467227173e+00    -6.80096e-03    1
 Kinetic       7.296282148690656e+00    -9.17586e-02    1
 N-E          -1.698162994947114e+01     9.13252e-02    1
 Coulomb       4.009360755731782e+00    -6.94761e-03    1
 Exchange      0.000000000000000e+00     0.00000e+00    1
 X-C          -1.667461383522072e+00     5.69460e-03    1
------------------------------------------------------------
 Electronic   -7.343537463603601e+00    -1.71956e-03    1
 Nuclear       2.756106540907633e-15     0.00000e+00    1
------------------------------------------------------------
 Total        -7.343537463603599e+00    -1.71956e-03    1
============================================================


################ Wall time: 2.29283e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    1.03866e-01    -7.297861382726141e+00   -7.29786e+00
  2    1.98237e-02    -7.341817902418927e+00   -4.39565e-02
  3    1.03324e-02    -7.343537463603599e+00   -1.71956e-03
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        4.07844e-03
 Diagonalizing matrix                           6.22090e-05
------------------------------------------------------------
                 Wall time: 8.13962e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     904      3.14373e-01
 Coulomb density                       456      3.56093e-01
 Coulomb potential                     200      1.91977e+00
 XC alpha density                      456      2.41748e-01
 XC beta density                       456      1.30401e-01
 XC evaluate xcfun                    1368      1.33694e+00
 XC energy                             456      2.47750e-05
 XC potential                          912      2.41369e-02
 XC grid size                          456          1097600
 XC grid change                       -448         -1075648
------------------------------------------------------------
                 Wall time: 4.33313e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 2.52322e+00 sec 
============================================================


============================================================
                 Calculating dipole moment
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.25838e+00 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-1.87569e+00  0.00000e+00  0.00000e+00
 0.00000e+00 -1.16112e-01  0.00000e+00
 0.00000e+00  0.00000e+00 -1.86793e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -1.875693400125915e+00
    1    a    1                      -1.867926244949046e+00
    2    b    1                      -1.161117912433313e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -3.859731436318292e+00
 Kinetic energy:                      7.255620526393152e+00
 E-N energy:                         -1.694169036010637e+01
 Coulomb energy:                      4.007724367649380e+00
 Exchange energy:                     0.000000000000000e+00
 X-C energy:                         -1.665259347228289e+00
 El. External field energy:          -1.160629790501528e-04
 Nuc. External field energy:          2.756106540907633e-15
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -7.343720876271178e+00
 Nuclear energy:                      2.756106540907633e-15
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.343720876271175e+00
                    (kJ/mol)         -1.928093637003604e+04
                    (kcal/mol)       -4.608254387553139e+03
                    (eV)             -1.998328164639383e+02
                                                            
============================================================
                                                            

                                                            
============================================================
                         Dipole moment                      
------------------------------------------------------------
                                                            
 Length of vector:   (au)                  1.1606297905e-01
                     (Debye)               2.9500263956e-01
                                                            
------------------------------------------------------------
                                                            
  -2.7011295328e-12   -2.6707192924e-12    1.1606297905e-01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 1.268488e+02                ***
***                                                      ***
************************************************************


